app/src/styles/index.scss

```diff
@@ -6,10 +6,10 @@
   box-sizing: border-box;
 }
 
-:root {
-  // makes it so that 1rem == 10px, 1.4rem == 14px, etc.
-  font-size: 62.5%;
-}
+// :root {
+//   // makes it so that 1rem == 10px, 1.4rem == 14px, etc.
+//   font-size: 62.5%;
+// }
```

Comment:
The font-size thing was making everything really small. I was having trouble getting it to display in a sensible way with this (everything was really small) so I commented it out, but I'd like to chat about how this works so I can fix this up

@clhenrick

Reply:
Thanks for the context Chris. setting the font-size variable to 1.6rem didn't do anything, I think the root font-size arg is overriding it for the material-ui library.

I don't want to bikeshed over this right now, so filed some github issues for deciding about component libraries (https://github.com/SatcherInstitute/het-frontend/issues/10) and css styling (https://github.com/SatcherInstitute/het-frontend/issues/11)